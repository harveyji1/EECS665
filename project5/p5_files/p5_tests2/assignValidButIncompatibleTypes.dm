main : () void {
     f : bool = true;
    g : int = 5;
    f = g;
}
