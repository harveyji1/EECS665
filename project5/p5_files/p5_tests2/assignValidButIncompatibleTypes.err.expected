FATAL [4,5]-[4,10]: Invalid assignment operation
Type Analysis Failed
